hazelcast/vector.py

```diff
@@ -60,6 +96,7 @@ class SearchResult:
 
 
 class Metric(enum.IntEnum):
+    """Metric is the similarity metric to use for indexing a vector index."""
```

Comment:
This definition is somehow very convoluted. I am sure it can be improved but I am unsure how.

I propose to start with 
`Metric is the measure of similarity which is used for vector indexing to enable searching.`

The docs team can likely come up with something better

Reply:
Not sure your suggestion is more understandable than the previous one :)

Reply:
I agree my suggestion is not an improvement, I was waiting for the docs team to comment. We can pick this up later and merge as is for now